docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,210 @@
+   $ python2.7 greeter_server.py
+   ```
+
+2. In another terminal, run the client
+
+   ```sh
+   $ python2.7 greeter_client.py
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the
+server for the client to call. Our gRPC service is defined using protocol
+buffers; you can find out lots more about how to define a service in a `.proto`
+file in [What is gRPC?]() and [gRPC Basics: Python][]. For now all you need
+to know is that both the server and the client "stub" have a `SayHello` RPC
+method that takes a `HelloRequest` parameter from the client and returns a
+`HelloResponse` from the server, and that this method is defined like this:
+
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+Let's update this so that the `Greeter` service has two methods. Edit `examples/proto/helloworld.proto` and update it with a new `SayHelloAgain` method, with the same request and response types:
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // Sends another greeting
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+Don't forget to save the file!
+
+## Generate gRPC code
+
+Next we need to update the gRPC code used by our application to use the new service definition. From the `examples/python/helloworld` directory:
+
+```
+$ ./run_codegen.sh
+```
+
+This script invokes the protocol buffer compiler as:
+
+```
+python -m grpc.tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/helloworld.proto
+```
+
+This generates `helloworld_pb2.py`, which contains our generated client and server classes, as well as classes for populating, serializing, and retrieving our request and response types.
+
+## Update and run the application
+
+We now have new generated server and client code, but we still need to implement and call the new method in the human-written parts of our example application.
+
+### Update the server
+
+In the same directory, open `greeter_server.py`. Implement the new method like this:
+
+```
+class Greeter(helloworld_pb2.GreeterServicer):
+
+  def SayHello(self, request, context):
+    return helloworld_pb2.HelloReply(message='Hello, %s!' % request.name)
+
+  def SayHelloAgain(self, request, context):
+    return helloworld_pb2.HelloReply(message='Hello again, %s!' % request.name)
+
+...
+
```

Comment:
Consider dropping this blank line? Maybe it can stay if it's providing the effect that you want after rendering.